devextreme/devextreme.d.ts

```diff
@@ -1,11 +1,11 @@
-// Type definitions for DevExtreme 15.2.7
+// Type definitions for DevExtreme 15.2.9
 // Project: http://js.devexpress.com/
 // Definitions by: DevExpress Inc. <http://devexpress.com/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference path="../jquery/jquery.d.ts" />
 
-declare namespace DevExpress {
+declare module DevExpress {
```

Comment:
@vvakame we provide support for those who still work with version 1.0 of the compiler.